.github/workflows/docker.yml

```diff
@@ -21,24 +21,45 @@ on:
  push:
   branches:
   - main
```

Comment:
Also meant to ask about this. Do we need to check the docker builds again after merging? We could save some API usage by removing it, and it wont affect publishing the docker containers which is done with Cloud Build.

Reply:
My opinion is that we should remove it, to reduce overall steps (assuming the publish flow does a fresh build after merge).

Reply:
I agree. 👍